actionpack/lib/action_controller/railtie.rb

```diff
@@ -32,6 +33,10 @@ class Railtie < Rails::Railtie # :nodoc:
       ActionController::Helpers.helpers_path = app.helpers_paths
     end
 
+    initializer "action_controller.set_allowed_redirect_hosts" do |app|
+      config.action_controller.allowed_redirect_hosts = []
```

Comment:
I think you want to set the class attribute in the initializer, you've already made the config an empty array above (line 19). Right?